schemas/telemetry/health/health.4.schema.json

```diff
@@ -91,11 +91,25 @@
           "type": "object"
         },
         "pingDiscardedForSize": {
-          "maxProperties": 10,
-          "minProperties": 1,
           "properties": {
+            "<unknown>": {
```

Comment:
@acmiyaguchi From the bq schema diff, it looks like this field with name "<unknown>" just gets dropped and is absent from the bq schema. Does that sound like intended behavior?

I realize that generating a sane bq-compatible field name for "<unknown>" is a bit of a stretch, but I was expecting that this might be normalized to "_unknown_".

Reply:
No, there is currently no behavior for normalizing an empty string as a key into BigQuery. This is certainly something that may be added, perhaps mapping it to `_` or `__unknown__`. However, this key would conflict with other JSON keys like `.`, `_`, or `__`. I think the right behavior would be to panic or drop the fields in the case where the transpiler detects these name clashes. 

Would this be handled deterministically in the decoder?

```javascript
>> x = {}; x[""]=1; x["."]=2; x["_"]=3;
3
>> x
Object { "": 1, ".": 2, _: 3 }
```

Reply:
I'm actually not sure why "<unknown>" is converted into "", that's unintended behavior for sure.

Reply:
Redash was converting "<unknown>" to "". The BQ console shows the value as "<unknown>". So I would expect the schema generator to be seeing "<unknown>" as the input name and potentially normalizing to "_unknown_".

Reply:
Yeesh, having this discussion is difficult due to literally typing `<unknown>` getting interpreted as an html tag and hidden.

Reply:
I was able to reproduce this locally in a shell, outside of any rendering done by javascript/browser. 

```
echo '{"properties": {"payload": {"properties": {"<unknown>": {"type": "string"}, "foo": {"type": "string"}}}}}' | jq                               
{
  "properties": {
    "payload": {
      "properties": {
        "<unknown>": {
          "type": "string"
        },
        "foo": {
          "type": "string"
        }
      }
    }
  }

python % echo '{"properties": {"payload": {"properties": {"<unknown>": {"type": "string"}, "foo": {"type": "string"}}}}}' | jsonschema-transpiler -t bigquery               
[
  {
    "fields": [
      {
        "mode": "NULLABLE",
        "name": "foo",
        "type": "STRING"
      }
    ],
    "mode": "NULLABLE",
    "name": "payload",
    "type": "RECORD"
  }
]
```